app/views/puppetclass_lookup_keys/welcome.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for(:title, _("Smart Class Parameters")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("info-circle", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Smart Class Parameters') %></h1>
+  <p><%= _("Parameterized class support permits detecting, importing, and supplying parameters direct to classes which support it, via the ENC. This requires Puppet 2.6.5 or higher.") %></p>
```

Comment:
1. I don't think specifying Puppet 2.6.5 or higher is necessary any more.
1. This should mention something about the matchers capability (or "rules" as you've called them in the smart variable page).